TheWarWithin/DruidBalance.lua

```diff
@@ -2874,7 +2888,7 @@ spec:RegisterAbilities( {
 
 
     starsurge = {
-        id = function() return state.spec.balance and 78674 or 197626 end,
```

Comment:
This is the only version of Starsurge that's in the Druid files; theoretically, other specs can use Starsurge under come conditions, no?

Reply:
I'll check, it may be that I need to bump the resto version up into feral. The specs go alphabetically by filename when trying to call registrations from other specs, right?

Reply:
I think it goes in load order, with current spec superceding.  Which I think tends to be alphabetical order.

Reply:
I've copied it onto Guardian and Feral, their versions most closely match the Resto version, and the talent checks take care of the rest anyways. And they both use the resto spell ID, Balance is the odd one out.